.github/workflows/local-deployment.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       run: |
         set -euxo pipefail
         juju add-model jenkins
-        juju deploy ch:jenkins --series=focal --config password=password --config plugins=jenkins-multijob-plugin
+        juju deploy ch:jenkins --series=noble --config password=password --config plugins=jenkins-multijob-plugin
```

Comment:
This an old skool juju? We should be using base here?

Reply:
Need to work with 3.5